flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -105,6 +105,9 @@ var A320_Neo_UpperECAM;
             this.iceNotDetTimer1 = new NXLogic_ConfirmNode(60);
             this.iceNotDetTimer2 = new NXLogic_ConfirmNode(130);
             this.predWsMemo = new NXLogic_MemoryNode(true);
+            this.showHydPtuMemoTimer = new NXLogic_ConfirmNode(0);
```

Comment:
the `NXLogic_ConfirmNode` parameter is a the amount of time in ms it takes until the state becomes true.

So I am actually not sure whether a confirmation node is actually needed, but if so, it probably should not be 0ms and 2ms.

Reply:
The pr is not yet ready for testing and it's not labeled as such.